apps/cowswap-frontend/src/common/containers/InvalidLocalTimeWarning/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { GlobalWarning } from '@cowprotocol/ui'
+
+import { useLocalTimeOffset } from './localTimeOffsetState'
+
+const TIME_OFFSET_THRESHOLD = 60 // 60 seconds
+
+/**
+ * When the local device time is not valid ()
+ */
+export function InvalidLocalTimeWarning() {
+  const localTimeOffset = useLocalTimeOffset()
+
+  if (!localTimeOffset || localTimeOffset < TIME_OFFSET_THRESHOLD) return null
+
+  console.debug('Local time offset:', localTimeOffset)
+
+  return (
+    <GlobalWarning>
+      Local device time does not match real time, CoW Swap most likely will not work correctly. Please synchronize the
+      time settings on your device with the real time.
```

Comment:
Sentence could be shorter.
Maybe:

```suggestion
      Local device time does is not accurate, CoW Swap most likely will not work correctly. Please adjust your device's time.
```

Not very happy with that either.

@MindyCoW ideas?